device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/BALANCED/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,36 @@
+{%- set default_cable = '5m' %}
+
+{%- include 'buffer_ports.j2' %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
```

Comment:
instead of lossless pool, should we say this is ingress_pool?

Reply:
currently, the qos tests are using these names for grabbing the configurations, so changing the name will cause them to break.